docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -1,8 +1,166 @@
 
 # Restaking Developer Guide
 
-Direct smart contract Restaking allows users to interact with the public smart contracts for a more customized Restaking experience. Please find the following developer resources to get started:
+Smart Contract Restaking allows the user to interact directly with the EigenLayer core contracts. The following sections describe how to setup your Restaking integration with the EigenLayer contracts directly with no reliance on the EigenLayer Web App.
+
+The Smart Contract Restaking user guides below serve as a high level overview of the most important steps for required for Restaking.
+
+Please note the following resources are available for users that wish to take a deeper dive on the contracts:
+* [EigenLayer Protocol Developer Docs](https://github.com/Layr-Labs/eigenlayer-contracts/tree/dev/docs): detailed protocol specifications for restaking smart contract integration developers.
+* [EigenLayer Protocol Contract Addresses](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#deployments): deployed contract addresses for Mainnet and Testnet.
+* [EigenLayer Protocol Integration Tests](https://github.com/Layr-Labs/eigenlayer-contracts/tree/dev/src/test/integration): tests that serve as examples on how to interact with the EigenLayer core contracts.
+
+
+
+
+
+# Smart Contract Liquid Restaking User Guide
+
+## Steps to Deposit (Restake) Liquid Tokens
+
+1. Call StrategyManager.depositIntoStrategy() .
```

Comment:
For the token being deposited, the user needs to call `ERC20(token).approve(strategyManager, amount)` before depositing.

Reply:
good call! fixed now